apps/condo/domains/ticket/components/TicketStatusSelect.tsx

```diff
@@ -58,7 +62,8 @@ export const TicketStatusSelect = ({ ticket, onUpdate, organization, employee, l
     const updateTicketStatus = useCallback((variables) => runMutation({
         action:() => update(variables, ticket),
         intl,
-    }), [ticket])
+        OnCompletedMsg: getSuccessfulChangeNotification,
+    }), [getSuccessfulChangeNotification, intl, ticket, update])
```

Comment:
I think you don't need useIntl hook result as dependency here
```suggestion
    }), [getSuccessfulChangeNotification, ticket, update])
```